src/index.html

```diff
@@ -6,6 +6,9 @@
 	<base href="/">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<link rel="icon" type="image/x-icon" href="favicon.ico">
+	<script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
```

Comment:
Removed jquery and isotope. Implemented grid using Masonry with image. dfabf925e6aae43fb6994d2f81e72cbeea391d80